### PR TITLE
Remove tmp-promise

### PR DESCRIPTION
**Pull request by DavidIsa**
Removing unnecessary dependency tmp-promise

### CODE REVIEW

File: src/cp.ts [hunk 1]
```diff
@@ -65,7 +64,7 @@ export class Cp {
      * @param {string} containerName - The name of the container in the pod to exec the command inside.
      * @param {string} srcPath - The source path in local
      * @param {string} tgtPath - The target path in the pod
-     * @param {string} cwd - The directory that is used as the parent in the host when uploading
+     * @param {string} [cwd] - The directory that is used as the parent in the host when uploading
```
Comment (mstruebing) [inline src/cp.ts:67]: Why did you change the `cwd` to be in square brackets?
Doesn't that mean it's an array of strings while it isn't?

**Review by mstruebing — COMMENTED** [review]
Nice change, but could we use this function instead of writing our own with a UUID? 
[fsPromises.mkdtemp](https://nodejs.org/docs/latest-v14.x/api/fs.html#fs_fspromises_mkdtemp_prefix_options) It's available in node 14 and according to our workflow, this is the oldest node version we are supporting in `master`.

File: src/cp.ts (hunk 1 above)
Comment (DavidIsa) [reply src/cp.ts:67]: When a param name is in the square brackets, it means that it is optional

https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns

**Comment by DavidIsa** [comment]
[fsPromises.mkdtemp](https://nodejs.org/docs/latest-v14.x/api/fs.html#fs_fspromises_mkdtemp_prefix_options) creates a temporarily directory, but in the code we need to create temporarily file.

File: src/cp.ts (hunk 1 above)
Comment (mstruebing) [reply src/cp.ts:67]: Thank you I did not know this.

**Comment by mstruebing** [comment]
You are right, I'm sorry I misunderstood the code. 👍

**Review by mstruebing — APPROVED** [review]

**Comment by brendandburns** [comment]
One small nit, I know it is _very_ unlikely that the file already exists, but can we do a check to make sure (and generate a new random name if it does)
